lib/chef-acceptance/chef_runner.rb

```diff
@@ -74,19 +74,17 @@ def chef_config_template
           #{File.expand_path('..', acceptance_cookbook.root_dir).inspect},
           #{File.expand_path('../../../.shared', acceptance_cookbook.root_dir).inspect}
         ]
-        node_path #{File.expand_path('nodes', acceptance_cookbook.root_dir).inspect}
+        node_path "#{node_path}"
         stream_execute_output true
         audit_mode #{app_options.audit_mode ? ":enabled" : ":disabled"}
```

Comment:
I do not think we should. With this change we are creating a temp chef directory for each suite. If we put file cache under it, it will not be shared across the different acceptance runs anymore. I do not see any reason why they should not go into `/var/chef/cache`